query/common/src/main/java/org/codice/ddf/admin/common/fields/base/ListFieldImpl.java

```diff
@@ -83,6 +88,7 @@ public void setValue(List values) {
 
         for (Object val : values) {
             try {
+                // TODO: tbatie - 4/22/17 - Should we consider making a copy method that is enforced by the interface instead?
```

Comment:
The down side to this is every field needs to override the copy method. The upside is every field won't have to provide a default constructor. @peterhuffer thoughts?